bin/varnishd/mgt/mgt_param_tweak.c

```diff
@@ -239,17 +239,17 @@ fmt_bytes(struct vsb *vsb, uintmax_t t)
 	const char *p;
 
 	if (t == 0 || t & 0xff) {
-		VSB_printf(vsb, "%jub", t);
+		VSB_printf(vsb, "%juB", t);
 		return;
 	}
-	for (p = "kMGTPEZY"; *p; p++) {
```

Comment:
Do we really want to remove exa/zeta/yotta?
With #2929 this is probably right, but it still triggers my #3723 itch